apps/block_scout_web/lib/block_scout_web/controllers/address_contract_verification_controller.ex

```diff
@@ -45,6 +45,39 @@ defmodule BlockScoutWeb.AddressContractVerificationController do
     end
   end
 
+  def new(conn, _params) do
+    changeset =
+      SmartContract.changeset(
```

Comment:
minor: normally you'd pipe the module into the changeset to build a new one

like 

```elixir
changeset = %SmartContract{}
|> SmartContract.changeset(%{})
```